setup.py

```diff
@@ -53,7 +55,7 @@
     setup_requires=['setuptools-markdown'],
     install_requires=[
         "eth-utils>=1.0.2,<2",
-        "rusty-rlp>=0.1.15, <0.2",
+        "rusty-rlp>=0.1.15, <0.2" if is_64_bit else ""
```

Comment:
My spider sense says that logic like this doesn't work at installation time in certain contexts.  For example, I *think* that when installing with a wheel this file may not be evaluated?!???  but I'm really not sure about this.

Reply:
I'm not sure I understand. You mean the fact of having the ternary expression at this particular place or evaluating the `is_64_bit` in general? If the former, then would moving the `install_requires` out of the `setup` function change that? We certainly [build it dynamically in Trinity, too](https://github.com/ethereum/trinity/blob/2aa13a83fce515502c78cf7f8013ba192816e607/setup.py#L121-L134)

If you mean the whole evaluation of the `is_64_bit` would not work then I wouldn't really know how to provide a proper fallback at all.

Reply:
@pipermerriam Any update on this? I got two more requests [asking for ARM builds](https://github.com/cburgdorf/rusty-rlp/issues/17#issuecomment-701979389) (which I can not provide right now, too). The 64 bit check may not be sufficient to exclude these (since there are 64 bit ARM chips) but the principle for how to handle the fallback should be more or less the same.

Reply:
I meant relying on having a python expression in the `setup.py` file be reliably evaluated at installation time.  I think that it is possible that for wheels, it may be evaluated at the time the wheel is created, but not during installation...  but I'm not sure about this.

Reply:
>Avoids arbitrary code execution for installation. (Avoids setup.py)

Mmh..sounds like it. Then, maybe the best option would be to not depend on `rusty-rlp` at all but have pyrlp use it if it is importable. And then Trinity would list it as a dependency separately, hence activate it that way. Web3.py and friends would not use it unless it is explicitly installed by the user.

Reply:
@carver @pipermerriam Alright, I made the changes to make this optional. Trinity can add a dependency `rlp[rust-backend]` to get the performance benefit. Waiting for re-approval. If we merge this today or tomorrow I could potentially still cut a new release this week.